enaml/widgets/dialog.py

```diff
@@ -68,6 +71,16 @@ def exec_(self):
             self.activate_proxy()
         return self.proxy.exec_()
 
+    def open(self):
+        """ Launch the dialog as a modal window.
```

Comment:
You could specify that this call is non blocking to make the difference with exec_

Reply:
Done.